### ISSUE TITLE
multipel 'RobotModel' cannot have different alpha

### ISSUE THREAD

**Issue by christian-rauch** [ros-visualization/rviz#1426]
I have two `RobotModel` (A and B) added to RViz, which show the same robot model at different configurations (A, B). Both widget have separate settings for the Alpha value. However, if I change the alpha for one of the `RobotModel` (A), the other one changes (B) too:
![peekKXPD8Z](https://user-images.githubusercontent.com/8226248/65345167-83f48a00-dbd1-11e9-86b5-24f5912f455d.gif)

The alpha value is not updated in the parameter overview of the other widget B, but it seems as if both models share the same visual properties.

### Your environment
* OS Version: Ubuntu 18.04
* ROS Distro: Melodic
* RViz, Qt, OGRE, OpenGl version as printed by rviz:
```
rviz version 1.13.3
compiled against Qt version 5.9.5
compiled against OGRE version 1.9.0 (Ghadamon)
```

**Comment by rhaschke** [comment]
Duplicate of #1385. Fixed in #1387, which was released into 1.13.4.

**Comment by christian-rauch** [comment]
Thanks for fixing this. I didn't notice that duplicate bug since I only searched in the open issues.
Looking forward to a binary release :-)
